packages/client/src/miner/miner.ts

```diff
@@ -325,18 +325,6 @@ export class Miner {
               `Miner: Assembled block full (gasLeft: ${gasLimit - blockBuilder.gasUsed})`
             )
           }
-        } else if ((error as Error).message.includes('tx has a different hardfork than the vm')) {
```

Comment:
this is a case no longer required as we stopped throwing on tx have different hardfork than vm, but instead set it to vm's if vm's and block's hf match